File: TopGun/src/app/model/animal.ts
export class Animal {

    public id: number;
    public dateOfBirth: Date;
    public type: string;
    public name: string;
    public breed: string;
    public gender: string;
    public size: string;
    public description: string;
    public photo: string|any ;
    public ownerId: number;
    public ownerSocialSecurityNumber: string;
    public certificate: string|any ;

    constructor(data: Partial<Animal>) {
        Object.assign(this, data);
    }

    public static empty(): Animal {
        return new Animal({
            id: 0,
            dateOfBirth: new Date(),
            type: '',
            name: '',
            breed: '',
            gender: '',
            size: '',
            description: '',
            photo: null,
            ownerId: 0,
            ownerSocialSecurityNumber: '',
            certificate: null,
        });
    }

}

File: MiracleMileAPI/MiracleMileAPI/Model/Animal.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MiracleMileAPI.Model
{
    public class Animal
    {
        public int Id { get; set; }
        public DateTime DateOfBirth { get; set; }
        public string Type { get; set; }
        public string Name { get; set; }
        public string Breed { get; set; }
        public string Gender { get; set; }
        public string Size { get; set; }
        public string Description { get; set; }
        public string Photo { get; set; }
         public string Certificate { get; set; }
         public int OwnerId { get; set; }
        public string OwnerSocialSecurityNumber { get; set; }

    }
}
